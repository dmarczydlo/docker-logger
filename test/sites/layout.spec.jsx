import React from 'react';
import Layout from '../../src/layout';

describe('layout', () => {

    it('should be rendered', () => {
        const wrapper = global.shallow(<Layout />);
        expect(wrapper).toMatchSnapshot();
    });
});
