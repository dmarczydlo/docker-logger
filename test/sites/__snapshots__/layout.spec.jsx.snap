// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout should be rendered 1`] = `
<EventListener
  onResize={[Function]}
  target="window"
>
  <Index
    width={3}
  />
</EventListener>
`;
